server/src/main/java/com/unosquare/carmigo/util/JwtTokenUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.unosquare.carmigo.util;
+
+import static com.unosquare.carmigo.constant.AppConstants.SECRET_KEY;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtTokenUtils
+{
+    public String generateToken(final UserDetails userDetails)
+    {
+        final Map<String, Object> claims = new HashMap<>();
+        claims.put("admin", false);
```

Comment:
I would place these strings as final static variables.

Reply:
This line actually shouldn't be here. I was playing around with the payload. My next PR will be about what a user can access and then this will come to play.